### DISCUSSION TITLE
Do I need two licenses: a Mac and a Linux?

### DISCUSSION THREAD

**Discussion by BittiBrightlight** [openaudible/openaudible#745] [Q&A] [answered]
My Audible account is on my Mac but I am in process of eventually migrating all my computing to a Linux laptop.  It seems that as long as the conversion is done into the MP3 files from my Mac, I can just take that data and pull them on my Linux without the need for a Linux license?

**Comment by openaudible** [comment, marked as answer]
No.. Just one is fine. We keep track of how many audible accounts you use.. limited to 5 for "family use".
